### ISSUE TITLE
NoWrapDoc

### ISSUE THREAD

**Issue by chiefdeals** [primefaces/primeflex#191]
No Wrap shows Wrap example in codebase
![Screen Shot 2023-11-17 at 11 45 48 AM](https://github.com/primefaces/primeflex/assets/126732208/d4e449b5-00d0-4bee-90d0-c2bbaf47f849)

**Comment by Ruhiddin** [comment]
yes, [small but significant mistake](https://primeflex.org/flexwrap#nowrap).
